.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Summary
+
+<!-- Please reference the issue this PR addresses. -->
+Addresses issue #
+
+## Checklist
+
+- [ ] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
```

Comment:
This is often redundant. When a pull request is for a small tweak, it is annoying to have to create an issue first and then create a PR. A PR may also be addressing an reported in the support forums, so linking to the topic would be needed in that case.

Reply:
This is quite important now that the team is growing, that we have Product Manager support and finally having a definition and execution pipeline. The main reasons are:
- Issues will now have a moderator section which includes, AC, Implementation Brief, Changelog (used to automatically generate release changelog and props) as well as a demo section. PRs do not include these information which will be used for every changes in the future.
- PR's cannot go through the definition pipeline since the definition happens prior to starting the work. For example prioritization should be done prior to starting the work in a way that it makes sense for the product and more importantly, adjusting the work load for the given sprint (quite important when having multiple engineers, UX etc. on the project). Implementation Brief should also be done prior to starting the work to avoid numerous cases of the implementation requested to be completely changed during code review cause unnecessary waste of time.
- In the background, the team velocity report is pulled using GitHub API. It becomes a nightmare to pull issues **and** PRs as it creates duplicates.

While it sometimes requires extra effort for small PRs, ad hock code changes should never really be made to avoid regression and for all the reasons mentioned above, keeping in mind that the little extra effort require to create an issue decrease a significant level of effort for everyone else.

There are more reasons to always track issues and not PRs which I will cover when walking everyone through the guidelines.